src/Filter/Converters/DateTimeMaxInclusiveConverter.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Globalization;
+using System.Linq;
+
+namespace RimDev.Filter.Range
+{
+    /// <summary>
+    /// Handles Inclusive Date Ranges ([2019-01-01,2019-
```

Comment:
Did some of this comment get cut-off?

Reply:
I think I just had a brain fart and stopped typing.